README.md

```diff
@@ -1,12 +1,22 @@
 # Randomly Adding Anything
 
-#### This mod add randomly generated ores, tools, armors and weapons.
-#### More to come ;-) .
+## Why is this?
+After a while, Minecraft's boundaries are massive. From surviving to fighting, from exploring to building to travelling across dimensions; Minecraft has many things and the only thing limiting you is your imagination.
+However, once you learn it, it becomes kind of the same every time; always mine the same minerals, with always the same tools, always the same biomes.
 
-## RoadMap
+This mod turns Minecraft back to that unexplored place that we once loved to learn to play and were afraid to explore!
 
-- Multiple textures for tools, weapons and armors.
-- Random effects on tools, ...
-- Randomly generated trees, plants.
-- Material info screen with REI.
+## Features
+
+### Randomize Ores!
+
+Each time the world is generated, it will create a new set of minerals and gems to be mined, with different pros/cons for each tool or armor.
```

Comment:
The set of materials is not generated every time you create the world but is generated the first time you launch the game, if you want to generate a new one you have to edit the config file (config/raa.json5) and set "regen" to true. Then it will generate a new set after restarting your game

Reply:
Fixed as of f5a0b6e ! Added that bit of information to a FAQ tab.

I believe these generation settings could be world-bound (and would be MUCH cooler that way), but I see how that would be harder to implement.